test/test_actors.py

```diff
@@ -561,6 +561,30 @@ def test_actorcritic(device):
     ) == len(policy_params)
 
 
+@pytest.mark.skipif(not _has_transformers, reason="missing dependencies")
+def test_AVO_from_lmhead_model():
```

Comment:
should we not test the execution of this model?

Reply:
done